src/spec/ListOfTasks.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import ListOfTasks from '../task-list.js';
```

Comment:
- [ ] Make sure that tests are passing. I have got the CLI error as well. Please fix that. Looks like there is a parsing error, if in your system works fine then please send a screenshot as well to confirm it.

![Screenshot from 2021-09-02 01-57-28](https://user-images.githubusercontent.com/22434660/131740279-7065c376-67a1-4ad0-b793-82ab795cc75f.png)


![Screenshot from 2021-09-02 01-54-55](https://user-images.githubusercontent.com/22434660/131740060-801bed01-2de1-40ed-98ca-d94d3145c5c6.png)

Reply:
All test on green on local system as show on screenshot.png

Reply:
I ran the test on my local environment and all passed :confetti_ball: 
![Screenshot from 2021-09-02 01-01-00](https://user-images.githubusercontent.com/1642014/131751191-01dce866-25f7-4473-a8d2-9903e4ba335e.png)